packages/contractkit/src/wrappers/Exchange.test.ts

```diff
@@ -0,0 +1,55 @@
+import { newKitFromWeb3 } from '../kit'
+import { testWithGanache } from '../test-utils/ganache-test'
+import { ExchangeWrapper } from './Exchange'
+
+/*
+TEST NOTES:
+- In migrations: The only account that has cUSD is accounts[0]
+*/
+
+testWithGanache('Exchange Wrapper', (web3) => {
+  const ONE = web3.utils.toWei('1', 'ether')
+
+  const kit = newKitFromWeb3(web3)
+  let accounts: string[] = []
+  let exchange: ExchangeWrapper
+
+  beforeAll(async () => {
+    accounts = await web3.eth.getAccounts()
+    kit.defaultAccount = accounts[0]
+    exchange = await kit.contracts.getExchange()
+  })
+
+  test('SBAT check buckets', async () => {
+    const [buyBucket, sellBucket] = await exchange.getBuyAndSellBuckets(true)
+    expect(buyBucket).toBeBigNumber()
+    expect(sellBucket).toBeBigNumber()
+    expect(buyBucket.toNumber()).toBeGreaterThan(0)
```

Comment:
Should we expect specific values here?

Reply:
Similar comment in a bunch of places below.

Reply:
disagree.
The actual number is related to the migrationConfig.js in protocol. Not sure what it is, and not the intent of my test.

The intent, is to validate that we are parsing the output and returning BigNumbers, and that the call was successfull (it returned without errors).

So, not a test of the Exchange contract itself, but of the wrapper

wdyt?

Reply:
Yeah, I agree for the most part, but was just thinking about the fact that right now all we're testing is that we made a call that successfully returned two numbers. 

Should we/is there a way to test that `exchange` wraps around the correct function on the correct contract? Maybe that's being too pedantic.

Reply:
mmm... you're right, the test doesn't really know if we are **actually** calling the contract using the rpc api.

honestly, i don't much value in doing that test, since it seems costly, and not very valuable. Also, we kinda "know" that we are calling the contract in the other tests (sellGold for example).

To be puristic, you should make a unit test, that mocks the contract object, or the transport layer and checks that something is being pass there. That's actually not that hard to do, we can spy on the provider, but again, i don't know if that's really valuable